Gemfile.lock

```diff
@@ -1,25 +1,28 @@
 GEM
   remote: https://rubygems.org/
   specs:
-    CFPropertyList (3.0.0)
-    activesupport (4.2.11.1)
+    CFPropertyList (3.0.2)
+    activesupport (4.2.11.12)
```

Comment:
4.2.11.12 seems wrong @amorde

Reply:
Ugh we have run into this in our other repos, I don’t know why it keeps happening. @dnkoutso was the fix to manually edit the lock file?

Reply:
yes unfortunately :(